textflip/commandline.py

```diff
@@ -0,0 +1,19 @@
+import argparse
+import sys
+
+from .rotation import flip
+
+
+def flip_cli():
+    if not sys.stdin.isatty():
+        lines = input().splitlines()
```

Comment:
`input().splitlines()` doesn't make a lot of sense: `input()` terminates when it sees a linefeed.

Do this instead:

```python
for line in sys.stdin:
    print(flip(line))
```

or the previously mentioned solution with `fileinput` (then you won't have to handle the two cases seperately:

```python
import fileinput

def cli():
    for line in fileinput():
        print(flip(line))
```

This would also make it so you can apply it to several files at once (think `cat(1)`), but of course you would lose the ability to easily flip a single word.

Reply:
right, i went from `sys.stdin.read()` to `input()` but that was a derp on my end.

Reply:
Also, `for line in sys.stdin` seems to have issues with newlines that need to be trimmed, so `sys.stdin.read.splitlines()` seems to be cleaner and good enough. Thoughts?

Reply:
That will break/consume a lot of RAM on very large inputs.

When reading from stdin, the input will already contain the final newline, so instead of printing you can just directly write to stdout (`sys.stdout.write(flip(line))`).

Reply:
wait, that doesn't seem to be the issue. the issue seems to be that `flip("abc\n")` returns `"\nɔqɐ"` -- that's where the weird behaviour was coming from. I guess we'll have to trim the newline, or make changes to `flip` :)

Reply:
Right, flip blindly reverses the order of unknown characters.

I honestly don't know what would be the cleanest solution here. My gut tells me that we should handle this _here_, not inside the core functionality, but I might be wrong. What if you hand a multiline text to `flip()`; shouldn't that work as expected (i.e. keep the position of newlines instead of flipping them)? I have to think about this a bit :D